website_snippet_carousel_product/templates/snippet.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Modify Snippets Menu -->
+    <template
+        id="snippets"
+        inherit_id="website.snippets"
+        name="Snippet Carousel Products Panel Option"
+    >
+        <xpath
+            expr="//div[@id='snippet_feature']//div[hasclass('o_panel_body')]"
+            position="inside"
+        >
+            <t
+                t-snippet="website_snippet_carousel_product.s_carousel_product"
+                t-thumbnail="/website_snippet_carousel_product/static/src/img/snippet_carousel_product.png"
+            />
+        </xpath>
+    </template>
+    <!-- Snippet Carousel Product -->
+    <template id="s_carousel_product" name="Carousel Products">
+        <div class="oe_structure" />
+        <section class="s_product_carousel">
+            <div class="container">
+                <div
+                    class="js_carousel_product"
+                    data-template="website_snippet_carousel_product.s_carousel_product_items"
+                />
+            </div>
+        </section>
+    </template>
+    <template id="s_carousel_product_items">
+        <div class="product_carousel_slider oe_website_sale">
+            <input
+                type="hidden"
+                name="product_count"
+                t-if="objects"
+                t-att-value="len(objects)"
+            />
+            <div
+                t-att-id="uniqueId"
+                class="carousel slide o_not_editable"
+                data-ride="carousel"
+            >
+                <div class="carousel-inner">
+                    <t t-set="products_index" t-value="0" />
+                    <t t-foreach="objects" t-as="products">
+                        <div
+                            t-attf-class="carousel-item #{not products_index and 'active' or ''}"
+                        >
+                            <div class="row">
+                                <t t-foreach="products" t-as="product">
+                                    <div t-attf-class="col-md-#{int(12 / pps)}">
+                                        <div t-attf-class="oe_product #{' '.join(x.html_class for x in product.website_style_ids if x.html_class)}" >
+                                            <t t-call="website_sale.products_item" />
+                                        </div>
+                                    </div>
+                                </t>
+                            </div>
+                        </div>
+                        <t t-set="products_index" t-value="products_index + 1" />
```

Comment:
Also unneeded.

```suggestion
```

Reply:
Why? without this you will get all slides as "active".